spec/jobs/create_feed_item_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe Jobs::CreateFeedItem do
+  include_context "spec helper"
+
+  fab!(:user) { Fabricate(:user) }
+  fab!(:post) { Fabricate(:post, user: user) }
+
+  it 'creates a feed item on Salesforce lead object' do
+    user.custom_fields[::Salesforce::Person::LEAD_ID_FIELD] = "lead_123"
+    user.save!
+
+    ::Salesforce::FeedItem.any_instance.expects(:create!).once
```

Comment:
Same concern as https://github.com/discourse/discourse-salesforce/pull/4/files#r823371345

Reply:
Same. It has a dedicated spec file to test the `.create!` method.